molo/polls/tests/base.py

```diff
@@ -57,3 +59,6 @@ def setUp(self):
 
         self.mk_main2(title='main3', slug='main3', path=00010003)
         self.client2 = Client(HTTP_HOST=self.main2.get_site().hostname)
+
+    def make_choice(self, title='yes'):
+        return Choice(title=title, latest_revision_created_at=datetime.now())
```

Comment:
I don't think you need to add the date. it usually gets set automatically.

Reply:
In this case it's causing the build to fail because it doesn't get set by anything and molo expects it to exist for the cache key.